packages/storybook/src/theme.ts

```diff
@@ -161,6 +161,66 @@ const dialogs = {
   },
 }
 
+const baseMessageStyles = {
+  bg: baseColors.gray[8],
+  color: baseColors.white,
+  fontSize: 1,
+}
+
+const messages = {
+  ...baseMessageStyles,
+  warning: {
+    ...baseMessageStyles,
+    borderLeftColor: baseColors.yellow[2],
+  },
+  danger: {
+    ...baseMessageStyles,
+    borderLeftColor: baseColors.red[2],
+  },
+}
+
+const baseBannerMessageStyles = {
+  ...baseMessageStyles,
+  borderRadius: '0',
+  borderColor: 'primary',
+  borderStyle: 'solid',
+  borderWidth: '2px',
+  display: 'flex',
+  justifyContent: 'center',
+  lineHeight: '14px', // Need to come up with something better here that is based on the scale
```

Comment:
Yea, I'm not too worried about this theme right now. I think in reality the theme is going to be pretty big but we are also adding variants for everything due to Storybook needed to document everything. This theme is still best used for testing purposes so I'm not overly concerned with making it perfect. 

I do agree that we need to look at typography so that we can deal with line-height and such. I would prefer to use a relative unit but want to make sure we are ready for that since we use only `px` right now.